jsapp/js/dataInterface.es6

```diff
@@ -434,6 +441,13 @@ export var dataInterface;
 
       return this._searchAssetsWithPredefinedQuery(params, query);
     },
+    searchMyCollectionMetadata(params = {}) {
+      return this._searchMetadataWithPredefinedQuery(
+        params,
+        // we only want the currently viewed collection's assets
+        `${COMMON_QUERIES.get('qbtc')} AND parent:${params.uid}`,
```

Comment:
Shouldn't this also (same as in `searchMyCollectionAssets`) be `parent__uid:` instead of `parent:`?